src/main/resources/db/migration/V010__CREATE_ROLE.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE role
+(
+    id      BIGINT          NOT NULL,
+    role    VARCHAR(255)    NOT NULL    UNIQUE,
```

Comment:
How come the roles are unique?

Reply:
The roles table consists of the available roles that can be assigned to the users without any direct associations to the users. admin, author, user, etc. Duplicates would not make sense here.

Reply:
OK, that makes sense. Thanks